api/XMLHttpRequestEventTarget.json

```diff
@@ -61,7 +61,7 @@
               "version_added": null
             },
             "edge": {
-              "version_added": null
+              "version_added": true
```

Comment:
This and the other onfoo attributes here confirmed by https://software.hixie.ch/utilities/js/live-dom-viewer/saved/6832 in Edge 18.